IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -158,16 +158,10 @@ - (void)setupCancelButton
 {
     UIBarButtonItem *cancelButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemCancel
                                                                                   target:self
-                                                                                  action:@selector(onCancel:)];
+                                                                                  action:@selector(userCancel)];
     self.navigationItem.leftBarButtonItem = cancelButton;
 }
 
-// Authentication was cancelled by the user
-- (IBAction)onCancel:(__unused id)sender
```

Comment:
just confirming if this was changes also in the subclasses, because I didn't see onCancel changes in the subclasses :)